rktlet/runtime/helpers.go

```diff
@@ -266,7 +267,15 @@ func generateAppAddCommand(req *runtimeApi.CreateContainerRequest, imageID strin
 	}
 	// TODO(yifan): logpath
 	// TODO(yifan): privileged
-	// TODO(yifan): mount, volumes
+
+	for _, mnt := range config.GetMounts() {
+		if mnt == nil {
+			glog.Warningf("unexpected nil mount: %v, %+v", mnt, config)
+			continue
+		}
+		volumeName := uuid.NewUUID()
+		cmd = append(cmd, fmt.Sprintf("--mnt-volume=name=%s,kind=host,source=%s,target=%s,readOnly=%t", volumeName, mnt.GetHostPath(), mnt.GetContainerPath(), mnt.GetReadonly()))
```

Comment:
While we don't support setting full propagation mode in rkt (yet?), we have support for recursive mode. Do we want/like to wire it in anyway here?